### ISSUE TITLE
Filter by grouped Or/And conditions

### ISSUE THREAD

**Issue by olragon** [tailflow/laravel-orion#126]
Can we implement this usage pattern?

```json
{
    "filters" : [
        {"operator": "or", "value": [
            {"field" : "created_at", "operator" : ">=", "value" : "2020-01-01"},
            {"operator": "and", "value": [
                {"field" : "meta.source_id", "operator" : "in", "value" : [1,2,3]}, 
                {"field" : "created_at", "operator" : ">=", "value" : "2019-01-01"},
            ]}
        ]}
    ]
}
```

```sql
WHERE created_at >= "2020-01-01" OR ( meta.source_id IN (1,2,3) AND created_at >= "2019-01-01" )
```

**Comment by alexzarbn** [comment]
Hi @olragon,

It is currently not supported and there is no scheduled milestone for this feature yet. Leaving the issue open to see, if more developers would need that functionality.

**Comment by alexzarbn** [comment]
The feature is now available as part of the [v2.13.0](https://github.com/tailflow/laravel-orion/releases/tag/2.13.0) release ✨
